src/redis-cli.c

```diff
@@ -1928,7 +1928,8 @@ static void usage(void) {
 "                     Default time interval is 1 sec. Change it using -i.\n"
 "  --lru-test <keys>  Simulate a cache workload with an 80-20 distribution.\n"
 "  --replica          Simulate a replica showing commands received from the master.\n"
-"  --rdb <filename>   Transfer an RDB dump from remote server to local file.\n"
+"  --rdb <filename>   Transfer an RDB dump from remote server to local file.
```

Comment:
```suggestion
"  --rdb <filename>   Transfer an RDB dump from remote server to local file.\n"
```
Don't need to change this line